sympy/utilities/tests/test_lambdify.py

```diff
@@ -775,11 +774,9 @@ def test_Indexed():
     assert lambdify(a, Sum(a[x, y], (x, 0, 1), (y, 0, 1)))(b) == 10
 
 def test_issue_12173():
-    exp1= lambdify((x,y), uppergamma(x,y),"mpmath")(1,2)
-    exp2= lambdify((x,y), lowergamma(x,y),"mpmath")(1,2)
-    exp1=simplify(exp1)
-    exp2=simplify(exp2)
-    ans1=(uppergamma(1,2)).evalf()
-    ans2=(lowergamma(1,2)).evalf()
+    exp1=N(lambdify((x,y), uppergamma(x,y),"mpmath")(1,2))
```

Comment:
It would be preferable to avoid importing N if that can be avoided. This line could be
```
    exp1 = lambdify((x, y), uppergamma(x, y), "mpmath")(1, 2)
```
(Note the space on either side of = and after each comma.)